### ISSUE TITLE
Error while sending a Signal to an Interface

### ISSUE THREAD

**Issue by guillemcordoba** [holochain/holochain#1068]
In particular workflows, `emit_signal` throws an error: 

```
        ☯ 
        21:50:35 [tryorama] error: Test error: {
          type: 'error',
          data: {
            type: 'ribosome_error',
            data: 'Wasm error while working with Ribosome: Host("Error while sending a Signal to an Interface: SendError(App(CellId(DnaHash(uhC0kT5JYvhlReulomid2JSz6KvbgTLyZUTfTQ2lGkgQVj7SfoGsY), AgentPubKey(uhCAk2fa_1dwToQLGBBPYx0ATlzjafKSzMohXtpvotdzHrOp-vG4d)), AppSignal(ExternIO([132, 164, 116, 121, 112, 101, 173, 78, 101, 119, 71, 97, 109, 101, 82, 101, 115, 117, 108, 116, 177, 97, 114, 101, 95, 108, 105, 110, 107, 115, 95, 109, 105, 115, 115, 105, 110, 103, 194, 170, 101, 110, 116, 114, 121, 95, 104, 97, 115, 104, 217, 53, 117, 104, 67, 69, 107, 75, 77, 70, 65, 57, 103, 105, 82, 86, 77, 105, 56, 110, 113, 69, 112, 111, 106, 76, 112, 66, 81, 80, 52, 89, 106, 57, 71, 110, 103, 49, 112, 101, 122, 53, 106, 85, 72, 68, 86, 77, 55, 57, 65, 79, 87, 99, 122, 171, 103, 97, 109, 101, 95, 114, 101, 115, 117, 108, 116, 132, 168, 112, 108, 97, 121, 101, 114, 95, 97, 131, 174, 112, 1
```

This doesn't happen for all emit_signals, but it seems to be reliable for particular workflows, like [this one](https://github.com/eyss/elo/runs/3976429468?check_suite_focus=true#step:6:295).

I have seen this in both tryorama and when connecting from a UI.

**Comment by freesig** [comment]
The only way to get this error is for there to be no app interface. So maybe there isn't an app interface for the signal to go to? Or maybe it's dieing for some reason

**Comment by guillemcordoba** [comment]
This seems to be fixed with holochain v0.0.118 and tryorama 0.4.6.
